dot/rpc/modules/system.go

```diff
@@ -258,8 +257,8 @@ func (sm *SystemModule) LocalListenAddresses(r *http.Request, req *EmptyRequest,
 	return nil
 }
 
-// LocalPeerId Returns the base58-encoded PeerId fo the node.
-func (sm *SystemModule) LocalPeerId(r *http.Request, req *EmptyRequest, res *string) error {
+// LocalPeerID Returns the base58-encoded PeerId fo the node.
+func (sm *SystemModule) LocalPeerID(r *http.Request, req *EmptyRequest, res *string) error {
```

Comment:
this should stay as `LocalPeerId` to represent the RPC call `system_localPeerId` that is the name placed in the RPC specs -> https://polkadot.js.org/docs/substrate/rpc/#system

Reply:
done!